### ISSUE TITLE
TiDB Dashboard Conprof not working

### ISSUE THREAD

**Issue by breeswish** [pingcap/ng-monitoring#59]
1. Start ngm using 
   ```
   bin/ng-monitoring-server --pd.endpoints "127.0.0.1:2379"
   ```

2. Start a cluster using
   ```
   tiup playground v5.3.0 --tiflash 0 --without-monitor
   ```

3. Open TiDB Dashboard:

![image](https://user-images.githubusercontent.com/1916485/146534769-8ad179ad-7d05-4fac-8185-1bc0077cf25a.png)

**Comment by crazycs520** [comment]
It will be working in `v5.4.0` TiDB cluster. 

Look like `tidb-dashboard` should to do a bug-fix cherry pick to v5.3.0, which is no need to check Prometheus when check whether ngm deploy. cc @baurine PTAL

**Comment by baurine** [comment]
> It will be working in `v5.4.0` TiDB cluster.
> 
> Look like `tidb-dashboard` should to do a bug-fix cherry pick to v5.3.0, which is no need to check Prometheus when check whether ngm deploy. cc @baurine PTAL

I think we just need to update the latest dashboard to PD 5.3 release is fine, right?
